gcloud/pubsub/api.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2015 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+""" Define API functions (not bound to classes)."""
+
+
+def list_topics(page_size=None, page_token=None,
```

Comment:
Ah, ok:  yes, it is intentionally a free function like `datastore.get`.

FWIW:  I left out the possibility of having `list_topics` return actual `Topic` instances (rather than the almost-empty mapping describing the topic).  `list_subscriptions` (in #743) might also need to return `Subscription` instances (it would then need to create `Topic` instances too).